addon/components/polaris-form-layout/group.js

```diff
@@ -28,18 +35,58 @@ export default Component.extend({
    */
   condensed: false,
 
+  /**
+   * Form layout group title
+   *
+   * @property title
+   * @type {String}
+   * @default null
+   * @public
+   */
+  title: null,
+
+  /**
+   * Form layout help text
+   *
+   * @property helpText
+   * @type {String|Component}
+   * @default null
+   * @public
+   */
+  helpText: null,
+
   /**
    * @private
    */
   role: 'group',
 
+  'data-test-form-layout-group': true,
+
+  titleID: computed('title', function() {
+    if (isBlank(this.get('title'))) {
+      return;
+    }
```

Comment:
Do you think it's worth wrapping up this logic in the `idVariation` util to clean these CPs up? I can't see that we'd ever want to generate an ID variation if the ID isn't present.

Reply:
ooooh yeah, my bad! Plz ignore + continue as you were 😅